tests/e2e/high_availability/distributed_coords.py

```diff
@@ -2870,5 +2874,272 @@ def get_vertex_count():
     mg_sleep_and_assert(follower_data, show_instances_coord2)
 
 
+def test_all_coords_down_resume():
```

Comment:
great test, can you add one more where the majority of coordinators are down?

Reply:
we have it here: def test_force_reset_works_after_failed_registration_and_2_coordinators_down(), do you mean something like that? What would you additionally test?